features/basic_replication.feature

```diff
@@ -6,12 +6,12 @@ Feature: basic replication
     And postgres0 is a leader after 10 seconds
     And I start postgres1
     When I add the table foo to postgres0
-    Then table foo is present on postgres1 after 15 seconds
+    Then table foo is present on postgres1 after 20 seconds
 
   Scenario: check the basic failover
     When I kill postgres0
-    Then postgres1 role is the primary after 30 seconds
+    Then postgres1 role is the primary after 32 seconds
```

Comment:
In worst case it takes 30 seconds to expire `leader` key. Plus ~2 seconds for running all health-checks: https://github.com/zalando/patroni/blob/master/patroni/ha.py#L159